decidim-budgets/spec/forms/project_form_spec.rb

```diff
@@ -84,35 +84,33 @@ module Decidim::Budgets
     end
 
     context "with proposals" do
+      subject { described_class.from_model(project).with_context(context) }
+
       let(:proposals_component) { create :component, manifest_name: :proposals, participatory_space: participatory_process }
       let!(:proposal) { create :proposal, component: proposals_component }
 
+      let(:project) do
+        create(
+          :project,
+          component: current_component,
+          scope: scope,
+          category: category
+        )
+      end
+
       describe "#proposals" do
-        it "returns the available proposals in a way suitable for the form" do
-          expect(subject.proposals)
-            .to eq([[proposal.title, proposal.id]])
+        before do
+          project.link_resources([proposal], "included_proposals")
         end
 
-        it "does not return the draft proposals" do
```

Comment:
I noticed that "codecov/patch" test of this PR is not passing by a higher percentage:
![image](https://user-images.githubusercontent.com/9702463/86464599-1b296900-bd30-11ea-98e3-08d83a289312.png)
so maybe you could add some more tests please?
e.g., I saw you have removed the unpublished proposals checking here @ahukkanen , could you add again please?

Reply:
Yes, because we are no longer displaying the proposals in that view directly, so this test is no longer relevant regarding this particular case and module.

These should be addressed in the proposal picker test coverage should be updated to include this case but it's not the point of this PR. However, this concern does not have any test coverage currently in the core why there was no place to move them to:
https://github.com/decidim/decidim/blob/develop/decidim-proposals/app/controllers/concerns/decidim/proposals/admin/picker.rb

That concern should be tested separately using shared examples and applied to all places that display proposal pickers.

This same issue applies to all tests for views that have proposal pickers, not just this particular case.

Reply:
Ok, you're right @ahukkanen 
Very good job! :+1: